source/Octopus.Tentacle/Communications/HalibutInitializer.cs

```diff
@@ -104,12 +106,27 @@ void AddPollingEndpoints()
                 var serviceEndPoint = new ServiceEndPoint(pollingEndPoint.Address, pollingEndPoint.Thumbprint, halibutProxy, halibutTimeoutsAndLimits);
 
                 //Open multiple polling connections if the env var is set to a non-zero/negative number
-                var connectionCount = int.TryParse(Environment.GetEnvironmentVariable(EnvironmentVariables.TentaclePollingConnectionCount), out var count)
-                    ? count
-                    : 1;
+                var connectionCount = 1u;
+                if (uint.TryParse(Environment.GetEnvironmentVariable(EnvironmentVariables.TentaclePollingConnectionCount), out var count))
+                {
+                    log.InfoFormat("Requested polling connection count: {0}", count);
+                    connectionCount = count;
+                }
+
+                //Coerce the requested value as it might be outside our max & min
+                switch (connectionCount)
+                {
+                    case > MaximumPollingConnectionCount:
+                        log.InfoFormat("Maximum polling connection count: {0}", MaximumPollingConnectionCount);
+                        connectionCount = 8;
```

Comment:
Nit: Just set it to `MaximumPollingConnectionCount` so the change of the limit in the future can be done just in one place.

Reply:
Derp. That's what it was meant to be. I'll fix it in a following PR 🤦

Reply:
Fixing in #882